test.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+import json
+
+json_file = '7_Land_Belcher_369784.json'
+collection_file = 'collection.csv'
+
+with open(json_file) as data_file:
+    json_data = json.load(data_file)
+
+deck_number = json_data.keys()[0]
+deck_object = json_data[deck_number]
+list_of_card_groups = deck_object['list']
+
+
+def get_df_of_cards_in_type(card_type):
+    cards_df = pd.DataFrame(list_of_card_groups[card_type])
+    cards_df['card_type'] = card_type
+    cards_df_with_good_column_names = cards_df.rename(columns={'deck-col-qty': 'how_many_the_deck_needs', 'deck-col-card': 'name', 'deck-col-price': 'price'})
```

Comment:
Agree... I should change col names in the scraping file when i also make the change to get the paper price. Kept names because that's the names in the HTML table I scrape from.